Mathlib/SetTheory/ZFC/Rank.lean

```diff
@@ -188,7 +189,6 @@ theorem le_succ_rank_sUnion : rank x ≤ succ (rank (⋃₀ x)) := by
   rw [mem_sUnion]
   exists z
 
-@[simp]
```

Comment:
I've got no idea why this broke.

Reply:
I'll reword this statement to use `iSup` instead of `lsub` in a future PR, that will also allow this to be a `simp` lemma again.

Reply:
In what way did it break? Is it that simpNF says it's already provable, or that something tells you this lemma can't be simp, or something else?

Reply:
nevermind! thanks to my `lsub` refactor it now works just fine.